modules/renter/hostdb/persist.go

```diff
@@ -1,37 +1,23 @@
 package hostdb
 
-import (
-	"log"
-	"os"
-	"path/filepath"
-
-	"github.com/NebulousLabs/Sia/persist"
-)
-
-const persistFilename = "hostdb.json"
-
-var saveMetadata = persist.Metadata{
-	Header:  "HostDB Persistence",
-	Version: "0.5",
+// hdbPersist defines what HostDB data persists across sessions.
+type hdbPersist struct {
+	Contracts []hostContract
 }
```

Comment:
Is this new, or has it been here? Because if the hostdb is doing persistence stuff, it should definitely be remembering activeHosts and allHosts